operator/validator/controller.go

```diff
@@ -698,10 +698,9 @@ func (c *controller) UpdateValidatorsMetadata(data map[spectypes.ValidatorPK]*be
 			if err != nil {
 				c.logger.Warn("could not start validator", zap.Error(err))
 			}
-			vc, found := c.validatorsMap.GetCommittee(v.Share().CommitteeID())
-			if found {
+			_ = c.validatorsMap.UpdateCommitteeAtomic(v.Share().CommitteeID(), func(vc *validator.Committee) {
```

Comment:
Shouldn't we log an error or something when `UpdateCommitteeAtomic` returns false here (just as a sanity check/thing) ?